Makefile

```diff
@@ -23,7 +23,7 @@ clean:
 	rm -rf vendor
 
 test: deps
-	@go test $(BUILDFLAGS) -cover $(PACKAGES)
+	@go test $(BUILDFLAGS) -cover $(PACKAGES) ./...
```

Comment:
not sure if it is the correct fix but `make test` doesn't run any test for me on master.  This is what I get:

```
$ make test
?   	github.com/projectatomic/skopeo/vendor/github.com/containers/image	[no test files]
```